src/amo/api/index.js

```diff
@@ -28,3 +28,16 @@ export function submitReview({
       }));
     });
 }
+
+export function getUserReviews({ userId } = {}) {
+  return new Promise((resolve) => {
+    if (!userId) {
+      throw new Error('userId cannot be falsey');
+    }
+    // Warning: if you rely on this API as is, you'll have to implement
+    // your own paging.
+    resolve(callApi({
+      endpoint: `accounts/account/${userId}/reviews`,
```

Comment:
I think it will just return the first page of results. I hope to replace it with https://github.com/mozilla/addons-server/issues/3986 asap though.